renderdoc/driver/gl/gl_manager.cpp

```diff
@@ -27,10 +27,14 @@
 #include <algorithm>
 #include "driver/gl/gl_driver.h"
 
-GLResourceManager::GLResourceManager(WrappedOpenGL *driver)
-    : ResourceManager(), m_Driver(driver), m_SyncName(1)
+GLResourceManager::GLResourceManager(CaptureState &state, WrappedOpenGL *driver)
```

Comment:
Notice, I made the `state` as the first parameter to be consistent with other managers which already accept it as the first argument.